nba_api_client.py

```diff
@@ -0,0 +1,42 @@
+from nba_api.stats.endpoints import leaguegamefinder
+from nba_api.stats.library.parameters import SeasonTypeAllStarNullable
+from nba_api.stats.endpoints import playbyplayv2
+import time
+
+
+class NbaApiClient:
+
+    def __init__(self, from_date, to_date, season_type=SeasonTypeAllStarNullable.regular, league_id='00'):
+        self.league_id = league_id
+        self.season_type = season_type
+        self.to_date = to_date
+        self.from_date = from_date
+
+    def retrieve_and_return_games_df(self):
```

Comment:
In this case, why not make `self._retrieve_games()` public and use that instead?